spot-ml/src/main/scala/org/apache/spot/SuspiciousConnects.scala

```diff
@@ -43,16 +44,22 @@ object SuspiciousConnects {
         val sqlContext = new SQLContext(sparkContext)
         implicit val outputDelimiter = config.outputDelimiter
 
+        val inputDataFrame = InputOutputDataHandler.getInputDataFrame(sqlContext, config.inputPath, logger)
+          .getOrElse(sqlContext.emptyDataFrame)
+        if(inputDataFrame.rdd.isEmpty()) System.exit(0)
```

Comment:
oh, you're saying that the exit automatically puts the error to logger.ERROR ?

I'd just like some nice human readable error messages, if that readable than I am good with it